### DISCUSSION TITLE
Virus report for pyqt

### DISCUSSION THREAD

**Discussion by theaspect** [Homebrew/discussions#1491] [Casks]
The problem is the same for me
https://www.reddit.com/r/homebrew/comments/nanj5b/found_malware_in_bottle_but_my_hackerone_ticket/

This is what I saw when did brew update
An infected file attempted to run on your device.
Threat name: Gen:Variant.Application.IOS.ZergHelperCA.1
Path: /usr/local/Cellar/pyqt/6.0.3_1/lib/python3.9/site-packages/PyQt6/QtGui.abi3.so
We quarantined the file to prevent malicious commands from being executed on your device.

**Comment by SMillerDev** [comment]
Did you try compiling it from source to see if the bottle is comprised?

**Reply by theaspect** [reply]
No I didn't
